### PR TITLE
update incorrect Quaternion type declaration

### PR DESCRIPTION
**Pull request by trusktr**

### CODE REVIEW

File: src/math/Quaternion.d.ts [hunk 1]
```diff
@@ -117,15 +117,10 @@ export class Quaternion {
...
 		src1: number[],
-		stcOffset1: number,
+		srcOffset1: number,
 		t: number
-	): Quaternion;
-
-	/**
-   * @deprecated Use {@link Vector#applyQuaternion vector.applyQuaternion( quaternion )} instead.
-   */
-	multiplyVector3( v: any ): any;
```
Comment (Mugen87) [inline src/math/Quaternion.d.ts]: Why have you removed the declaration of `multiplyVector3()`?

Comment (trusktr) [reply src/math/Quaternion.d.ts]: It doesn't seem to exist in the implementation: https://github.com/mrdoob/three.js/blob/dev/src/math/Quaternion.js

Comment (Mugen87) [reply src/math/Quaternion.d.ts]: It's defined in `THREE.Legacy.js`. So the method is still accessible.

**Comment by Mugen87** [comment]
Closing due merge conflicts.

**Pull request closed without merging**